tests/migration_test.go

```diff
@@ -340,6 +340,13 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		return kvc.Spec.Configuration
 	}
 
+	getVmisNamespace := func(vmi *v1.VirtualMachineInstance) *k8sv1.Namespace {
+		namespace, err := virtClient.CoreV1().Namespaces().Get(context.Background(), vmi.Namespace, metav1.GetOptions{})
+		Expect(err).ShouldNot(HaveOccurred())
```

Comment:
@fossedihelm I think its better to use `expectWithOffset` here

Reply:
Thanks, done!